stellarisdashboard/parsing/timeline.py

```diff
@@ -72,6 +72,7 @@ def process_gamestate(self, game_id: str, gamestate_dict: Dict[str, Any]):
                 )
                 if config.CONFIG.debug_mode or isinstance(e, KeyboardInterrupt):
                     raise e
+            _shared_description_cache.clear()
```

Comment:
I'm curious how you found what was leaking memory?

And remind me of the flow please - is the cache built while parsing a save, then this line clears it after one save is parsed, to free that memory?

Reply:
> I'm curious how you found what was leaking memory?

Nothing fancy. I'm remembered that was a relatively recent change, so I tried disabling the cache entirely, and that fixed the memory leak. Then I experimented to find a way to keep the cache (since that significantly increased speed) but not leak memory

The cache is gradually built up as the function is used, then cleared when we're done processing a save. I'll add a comment here